### PR TITLE
sorting algorithms for review sort options

### PR DESCRIPTION
**Pull request by etcaples**
The sorting helpers are not DRY, I'm going for mvp functionality right now - I've made a ticket to refactor my sort helpers later.

I also need to refactor my ReviewList component on a different working branch to allow for user input when trying to sort by measurement matches, so please disregard the code inside that helper for now.

(https://trello.com/c/Jx7Nv4NC/41-decide-what-featured-means-for-review-sorting)
(https://trello.com/c/BzKibd77/40-need-to-be-able-to-sort-reviews-by-newest-featured-rating-and-women-like-me)

### CODE REVIEW

File: client/dist/stylesheet.css [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+
+.fit-summary_bar-highlight {
+  background-color: #C09566;
+  height: 10px;
+  width: 17px;
+  transition: width .2s ease-in-out;
+}
+
+.fit-summary_count .label {
+
```
Comment (frokrab) [inline client/dist/stylesheet.css]: You can probably get rid of this empty object!

File: client/dist/stylesheet.css [hunk 2]
```diff
@@ -0,0 +1,310 @@
...
+.review-filters-wrapper {
+  width: 990px;
+  padding: 20px 0;
+  margin: 50px auto 0 auto;
+}
+
+.review-filters {
+  width: 35%;
+  /*float: left;*/
```
Comment (frokrab) [inline client/dist/stylesheet.css]: Pull out any comments you don't need anymore!

File: client/src/components/App.jsx [hunk 3]
```diff
@@ -8,23 +8,34 @@ class App extends React.Component {
...
+    this.sortByMeasurements = this.sortByMeasurements.bind(this);
   }
 
   componentDidMount() {
     this.getUserData();
   }
 
   getUserData() {
-    axios.get('/reviews/1')
+    axios.get('/reviews/21')
```
Comment (frokrab) [inline client/src/components/App.jsx]: For production your component should render a specific item page based on an id in the address bar. It sounds more difficult than it is-- let me know if you ned any help with this!

File: client/src/components/App.jsx [hunk 4]
```diff
@@ -57,11 +68,102 @@ class App extends React.Component {
...
+    } else if (this.state.counter === 1) {
+      // sorted = this.sortByMeasurements(data);
+    } else if (this.state.counter === 2) {
+      sorted = this.sortByFeatured(data);
+    } else if (this.state.counter === 4) {
+      sorted = this.sortByRating(data);
+    }
+
+    if (this.state.counter === 0 || this.state.counter === 1) {
+      this.setState({showFilterForm: true}, this.setState({data: sorted}));
```
Comment (frokrab) [inline client/src/components/App.jsx]: You can use one setState for both properties here, like so:
`this.setState({
  showFilterForm: true,
  data: sorted,
});`

File: client/src/components/ReviewFilter.jsx [hunk 5]
```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+const ReviewFilter = (props) => {
```
Comment (frokrab) [inline client/src/components/ReviewFilter.jsx]: For arrow functions you can omit the parenthesis around the arguments if there's only one argument

File: client/src/components/ReviewFilter.jsx [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+  let showFilterForm = props.showFilterForm;
+  let heights = props.heights.map((height) => {
```
Comment (frokrab) [inline client/src/components/ReviewFilter.jsx]: Same as above-- you don't need parentheses here

File: client/src/components/App.jsx (hunk 3 above)
Comment (etcaples) [reply client/src/components/App.jsx]: @frokrab yeah, when you have a minute I could use some help - thanks!

**Comment by etcaples** [comment]
Hey @frokrab I made the changes you suggested, please review. :)

**Comment by etcaples** [comment]
added some commits that resolve the following tickets:

(https://trello.com/c/IlmeQjkG/35-change-rating-numbers-to-star-ratings-http-schemaorg-rating-xl)
(https://trello.com/c/7tDgdI1H/44-make-the-fit-keyword-bars-reflect-how-users-rated-the-fit-change-number-of-reviews-to-percentage-of-reviews)
(https://trello.com/c/V2Vbjf92/31-fix-height-display-to-show-inch-and-foot-designations-instead-of-being-hyphenated-sm)

File: client/dist/stylesheet.css [hunk 7]
```diff
@@ -0,0 +1,391 @@
...
+
+.rev-stars-1 {
+  background: url('./sprite.css-abf57cf5.svg') no-repeat;
+  background-position: 53.56679636835279% 26.410564225690276%;
+  width: 89px;
+  height: 16px;
+}
+
+/*.rev-stars-0 {
+  background: url('https://sf-p.rtrcdn.com/images/svg/sprite.css-abf57cf5.svg') no-repeat;
```
Comment (frokrab) [inline client/dist/stylesheet.css]: Pull out this comment if you're done with it!

File: client/dist/stylesheet.css [hunk 8]
```diff
@@ -0,0 +1,391 @@
...
+  height: 10px;
+  transition: width .2s ease-in-out;
+}
+
+/*//////////////////////\\\\\\\\\\\\\\\\\\\\\*/
+/* PHOTO CAROUSEL */
+/*//////////////////////\\\\\\\\\\\\\\\\\\\\\*/
+
+.photo-carousel-wrapper {
+
```
Comment (frokrab) [inline client/dist/stylesheet.css]: You can probably get rid of this empty object here

**Pull request merged**